runtime-enviroment/src/code.py

```diff
@@ -15,6 +15,6 @@
 
 from time import sleep
 
-for i in range(100):
+for i in range(20):
```

Comment:
po co to?

Reply:
To jest symulowanie wykonywania kodu od klienta